core/page-assets/loader.php

```diff
@@ -75,6 +81,9 @@ public function add_assets( array $assets ) {
 		$this->assets = array_replace_recursive( $this->assets, $assets );
 	}
 
+	/**
+	 * @deprecated 3.22.0
+	 */
```

Comment:
@KingYes - Not sure I understand the latest changes related to the loader, I see that you've restored the call to enqueue_assets(), but why not go with the direction that you need?:
- enqueue_assets - will enqueue immediately (and not deprecated).
- enable_assets - will point to the enqueue_assets() and will be marked as deprecated.
- add_assets - will also register the assets.
In other words to get rid of the "pre-enabled" capability if it's not needed?